Support/PowershellSyntax.tmLanguage

```diff
@@ -324,13 +324,19 @@
 			<string>keyword.operator.other.powershell</string>
 		</dict>
 		<dict>
-			<key>comment</key>
-			<string>This is very imprecise, is there a syntax for 'must come after...' </string>
 			<key>match</key>
-			<string>(?&lt;!\s|^)\.\.(?=\d|\(|\$)</string>
+			<string>\.\.</string>
 			<key>name</key>
 			<string>keyword.operator.range.powershell</string>
 		</dict>
+		<dict>
+			<key>comment</key>
+			<string>Properties and methods</string>
+			<key>match</key>
+			<string>\.(\w)+</string>
```

Comment:
This separates properties and methods from variables. We can probably simplify the ```variable``` section some if we decide to stick with this method.